ansible/roles/etcd/tasks/start.yml

```diff
@@ -1,6 +1,2 @@
 ---
-- include: etcd-start.yml
-  when: not is_coreos
-
-- include: etcd2-start.yml
-  when: is_coreos
+- include: "{% if not is_coreos %}etcd-start{% else %}etcd2-start{% endif %}.yml"
```

Comment:
You mean refactoring all roles that include tasks like this to dynamic include? or just in 'etcd' role?